packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -239,7 +239,7 @@ var idxAuthClient = new IdxClient(null);
 var authnResponse = await idxAuthClient.VerifyAuthenticatorAsync(verifyAuthenticatorOptions, (IIdxContext)Session["idxContext"]);
 ```
 
-### Step 16: Handle response from VerifyAuthenticatorAsync
+### Step 16: Handle the submit response
```

Comment:
In the corresponding diagram, the steps are mis-numbered — after 15 goes straight to 17. 

Also, what about Step 18 (17?) — Show the remaining list of authenticators ?

Reply:
Fixed the numbering in the diagram. steps 15, 16, 17 are now aligned with the doc.  Step 18 is in the next diagram.